_posts/code/rspec/2016-04-09-database-cleaner-setup.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: Database Cleaner with RSpec
+category: code
+description: "Setting up Database Cleaner with RSpec"
+---
+
+When writing tests for a Rails application there is a good chance you will need
+to create records in the database. To have reliable tests those records should
+be removed after every run. This would be a painful process to do manually however
```

Comment:
The sentence that starts on this line and runs to the end of the paragraph is a bit of a run on, consider some commas or splitting it out